src/mapOfPeople.js

```diff
@@ -40,7 +40,20 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const roomWithPeople = new Map();
+  let countSeconds = 0;
+
+  people.forEach(person => {
```

Comment:
```suggestion
  people.forEach((person, index) => {
```
use index, instead of count